src/index.html

```diff
@@ -22,6 +22,286 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#">
+          <img
+            src="./images/logo.png"
+            alt="moyo logo"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__item">
+          <li class="nav__link">
+            <a class="nav__a is-active">Apple</a>
+          </li>
+          <li class="nav__link">
```

Comment:
Fix classes everywhere and add blank line between the links
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active">Apple</a>
          </li>
          
          <li class="nav__item">
```